packages/styles/base.css

```diff
@@ -58,6 +58,10 @@ ul {
   padding: 0;
 }
 
+button {
+  font-family: 'Roboto', Helvetica, Arial, sans-serif;
```

Comment:
Since this is inherited/shared, we should use a css variable in with the font-family in [`variables.css`](https://github.com/dequelabs/cauldron/blob/develop/packages/styles/variables.css), something like `--base-font-family: 'Roboto', Helvetica, Arial, sans-serif`. 

That can then get applied for the existing `body` selector, and here for the `button`.

Reply:
This change is made